mission_control/navigator_alarm/navigator_alarm/alarm_helpers.py

```diff
@@ -25,20 +25,20 @@ def __init__(self):
         # For now, unused - future: Use for system reports
         # self.alarm_spawn_pub = rospy.Publisher('/alarm_spawn', Alarm, latch=True, queue_size=1)
 
-        # Get ROS name
-        self.node_name = rospy.get_name()
```

Comment:
I moved this down so that it does this later on when you actually add an alarm. I was having issues with having the alarm name being `untitled` (or something like that) when you created an alarm broadcaster in a non ros file (ie. an alarm handler).